tests/pytests/test_issues.py

```diff
@@ -563,10 +563,13 @@ def test_mod_4207(env):
   env.expect('FT.SEARCH', 'idx2', '*', 'NOCONTENT').equal([3, 'address:1', 'address:2', 'address:3'])
 
 def test_mod_4255(env):
+  env.skipOnCluster()
+  conn = getConnectionByEnv(env)
+
   env.expect('FT.CREATE', 'idx', 'ON', 'HASH', 'SCHEMA', 'test', 'TEXT').equal('OK')
 
-  env.expect('HSET', 'doc1', 'test', '1').equal(1)
-  env.expect('HSET', 'doc2', 'test', '2').equal(1)
+  conn.execute_command('HSET', 'doc1', 'test', '1')
+  conn.execute_command('HSET', 'doc2', 'test', '2')
```

Comment:
Why changing that?

Reply:
You are skipping on cluster so not sure why you change it...

Reply:
@MeirShpilraien 
It is a new test and I often use the conn. Syntax to avoid confusion. 
Would you like me to switch?

Reply:
No, just wonder why you changed it.